templates/secrets_st2chatops.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-{{- range $env, $value := .Values.st2chatops.env }}
+{{- range $env, $value := .Values.secrets.st2.st2chatops.env }}
```

Comment:
Yeah, could you revert this? The `.Values.secrets` stuff needs an overhaul in a separate PR.
```suggestion
{{- range $env, $value := .Values.st2chatops.env }}
```

Reply:
Following up: `.Values.secrets` was removed in #203